File: mygrpc/app/myapp_client.py
import grpc

import myapp_pb2
import myapp_pb2_grpc

def run():
    channel = grpc.insecure_channel('localhost:50051')
    stub = myapp_pb2_grpc.myappStub(channel)
    response = stub.sayHello(myapp_pb2.queryRequest(queryString="Shakeeb"))
    for resp in response.results:
        print(resp.query+",", resp.response)

if __name__ == '__main__':
  run()

File: mygrpc/app/myapp_client.js
var PROTO_PATH = __dirname + '/../protos/myapp.proto';

var grpc = require('grpc');
var myapp_proto = grpc.load(PROTO_PATH).myapp;

function main() {
    var client = new myapp_proto.myapp('localhost:50051',
                                         grpc.credentials.createInsecure());
    client.sayHello({queryString:"Shakeeb"}, function(err, response) {
        response.results.forEach(function(result) {
            console.log('Response recvd: '+ result.response);
        });
    });
}

main();
